components/core/src/clp_s/TimestampPattern.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include <chrono>
 #include <cstring>
+#include <string_view>
```

Comment:
Nit: if I am not mistaken, string is also missing from the include
```suggestion
#include <string>
#include <string_view>
```